sass/site/primary/_posts-and-pages.scss

```diff
@@ -20,10 +20,43 @@
 	display: inline;
 }
 
+.page,
+.single {
+
+	.entry-header {
+		background: $blue;
+		color: $white;
+		margin-bottom: 95px;
+		margin-left: calc(50% - 50vw);
+		max-width: none;
+		padding: 80px 40px 60px;
+		position: relative;
+		text-align: center;
+		width: 100vw;
+
+		&::after {
+			content: "";
+			background: url("#{$short_wave}") no-repeat center;
+			position: absolute;
+			left: 0;
+			bottom: -95px;
+			height: 95px;
+			width: 100vw;
```

Comment:
I think this isn't working exactly right, the wave cuts off after 1440px:

![Screen Shot 2019-04-23 at 11 22 58 AM](https://user-images.githubusercontent.com/541093/56594222-756e5c00-65ba-11e9-8504-931cc78d28f1.png)

Reply:
Oof. That's what I get for doing this on my 13 inch Macbook only.

Reply:
I took a different route with this and wanted to get your thoughts. Since it looked pretty good at 95px from 1440px wide down I did a media query for above 1440px. That way above 1440px the height can grow so I can use `cover` for the background.

Reply:
What do you think about using `contain` for the <1440px sizes so we always get the full wave?

Reply:
I do like the look of `contain` better than `cover`. Good call.

Reply:
I actually meant to use `contain` always, on smaller screens too– I've updated the PR with that change, and I think this is good to merge now 👍